lib/bootstrap_form/inputs/check_box.rb

```diff
@@ -50,40 +50,26 @@ def check_box_value(name, value)
       end
 
       def check_box_classes(name, options)
-        classes = [options[:class]]
-        classes << (options[:custom] ? "custom-control-input" : "form-check-input")
+        classes = [options[:class], "form-check-input"]
```

Comment:
Your code appears equivalent to the legacy code with respect to `options[:class]`. However, I'm wondering if a user passed in an array of classes, would the code work? In many places the user can provide an array of classes. (I think it's supported by the underlying Rails helpers.) Maybe we should make this work for the case where someone provides an array of class names in `options[:class]`. What do you think?

Reply:
I just confirmed for myself that the `class` option supports an array of strings. That's cool, didn't know that was supported. Good catch, I'll refactor to account for the option.

Reply:
I'm realizing now that this is why `#flatten` is called in the return value - so this already worked. That said, I'd rather this be clearer so I'll fix it nonetheless.